LibraBFT/Impl/OBM/ECP-LBFT-OBM-Diff/ECP-LBFT-OBM-Diff-1.agda

```diff
@@ -42,10 +96,17 @@ e_RoundState_processLocalTimeoutM e r =
 
 ------------------------------------------------------------------------------
 
-postulate -- TODO-1: e_EpochManager_doECP_waitForRlec
- e_EpochManager_doECP_waitForRlec
-  : EpochManager → EpochChangeProof
-  → Either ErrLog Bool
+e_EpochManager_doECP_waitForRlec : EpochManager → EpochChangeProof → Either ErrLog Bool
```

Comment:
Eventually, before proving anything about this function, we'll want it in `EitherD`.  I'm OK with leaving that until then, but maybe add a TODO-1?

Reply:
OK